### ISSUE TITLE
Invalid credentials on signup

### ISSUE THREAD

**Issue by kimjongbing** [majorpeter/atomic-tracker#11]
When I make an account and then try login it says invalid credentials. My account is present in the sqllite database so I'm not sure what's happening.

**Comment by majorpeter** [comment]
Hello @kimjongbing ,

Thanks for reporting. I was able to reproduce it with a clean install:

```sh
docker run --rm --name my-atomic-tracker -p 8080:8080 -v ./config:/config ghcr.io/majorpeter/atomic-tracker:latest
```

This is the stack trace in the log:
```
Exception at url: /api/auth/login
Error: Failed to serialize user into session
    at pass (/app/backend/node_modules/passport/lib/authenticator.js:296:19)
    at Authenticator.serializeUser (/app/backend/node_modules/passport/lib/authenticator.js:314:5)
    at /app/backend/node_modules/passport/lib/sessionmanager.js:33:10
    at Immediate._onImmediate (/app/backend/node_modules/express-session/session/store.js:54:5)
    at processImmediate (node:internal/timers:466:21)
```

I'm looking into it now.

**Comment by majorpeter** [comment]
So I fixed this issue back in April in this commit:
* [fixed passportjs serialization issue when google login not available](https://github.com/majorpeter/atomic-tracker/commit/893e0668621584a2c19553fa9d7864e9707b005b)

I just haven't released it with a tag, so `master` docker image was fixed, `latest` was outdated.
I released `v1.3.1` today, so pull the latest image and it should work!

(Let me know if there's another issue.)
